AlibudaLab/atomic-habit-builder
# feat: connect farcaster

## Summary

In this pull request we implement the feature to allow user to connect and disconnect Warpcast to the account in the profile page.

## Linked Issue

closed #83 

## Details

Achieved through 
- web/app/profile/components/QRCodeDialog.tsx
- web/app/profile/components/SignInButton.tsx

And added a new provider to load profile related information

## Verification Steps

Please test in web and mobile browser
- Should be able Connect to Farcaster
- Should be able to load Farcaster profile after reload
- Should be able to Disconnect to Farcaster
